packages/app/src/views/Settings.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="w-full flex flex-col">
+    <header class="p-8 flex flex-row items-center">
+      <Button back round icon="fas fa-times" class="h-7 w-7" />
+      <span class="ml-2 text-3xl">Einstellungen</span>
+    </header>
+    <main class="bg-secondary max-h-full overflow-y-auto flex-grow p-6">
+      <!-- Useranzeige -->
+      <div class="flex flex-col justify-center items-center text-white">
+        <span class="flex text-xl font-heading">
+          <span class="my-auto fab fa-spotify" />
+          <span class="ml-2">Nutzer:</span>
+        </span>
+        <span class="text-2xl font-semibold">{{ userEmail }}</span>
+      </div>
+
+      <!-- Lautsprecherauswahl -->
+      <LabeledInput class="mb-5 h-20 mt-10" label="Lautsprecher">
+        <Dropdown v-model="selectedSpeaker" :items="speakerList" :removeable="false" />
+      </LabeledInput>
+    </main>
+
+    <footer class="flex text-gray-800 py-5">
+      <span class="flex w-full text-xl">
+        <Button round icon="fas fa-caret-left" class="ml-4 my-auto h-12 w-12" />
+        <Button text="Speichern" center-text class="mx-2 px-3 py-2 text-2xl flex-grow" />
+        <Button
+          text="Abmelden"
+          center-text
+          class="mx-2 px-3 text-2xl flex-grow"
+          @click="logoutUser()"
+        />
```

Comment:
why is this in the footer? in my mind it would be way more intuitive if this was under the user name

Reply:
I will change that. To be honest i have struggled a bit with three diffrent versions of the figma layout ^^